Dockerfile-local

```diff
@@ -9,6 +9,7 @@ LABEL maintainer="csse-education-research@canterbury.ac.nz"
 ARG DEBIAN_FRONTEND=noninteractive
 
 ENV DJANGO_PRODUCTION=False
+ENV INCLUDE_INCONTEXT_L10N=True
```

Comment:
What happens when you view in-context mode on a branch with changes yet to make it to Crowdin?

Reply:
As discussed offline, this isn't an issue. Content changes in a branch won't affect the in-context l10n features on that branch at all. Backend changes, while they might make the in-context page on that branch unusable until merged, won't break anything.